spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkParquetReadMetadataColumns.java

```diff
@@ -104,7 +103,6 @@ public class TestSparkParquetReadMetadataColumns {
       }
       row.update(1, UTF8String.fromString("str" + i));
       row.update(2, i);
-      row.update(3, false);
```

Comment:
Why do we need these changes?

Reply:
They are broken by the vectorized changes. I added back them since we are going to handle the vectorized version in another PR.